src/PIL/Image.py

```diff
@@ -2170,11 +2172,16 @@ def show(self, title=None, command=None):
 
         On Windows, the image is opened with the standard PNG display utility.
 
-        :param title: Optional title to use for the image window,
-           where possible.
-        :param command: command used to show the image
+        :param title: Optional title to use for the image window, where possible.
         """
 
+        if command is not None:
+            warnings.warn(
+                "The command parameter is deprecated and will be removed in a future"
```

Comment:
```suggestion
                "The command parameter is deprecated and will be removed in a future "
```
When I run this code at the moment, I get future release as one word.
> DeprecationWarning: The command parameter is deprecated and will be removed in a futurerelease. Use a subclass of ImageShow.Viewer instead.